spi/common/identity-trust-spi/src/main/java/org/eclipse/edc/identitytrust/model/VerifiablePresentation.java

```diff
@@ -17,23 +17,21 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static org.eclipse.edc.identitytrust.VcConstants.VC_PREFIX;
+
 /**
  * Represents a VerifiablePresentation object as specified by the
  * <a href="https://w3c.github.io/vc-data-model/#presentations-0">W3 specification</a>
  */
 public class VerifiablePresentation {
-    public static final String DEFAULT_TYPE = "VerifiablePresentation";
-    private final String rawVp;
```

Comment:
Will there be a `VerifiablePresentationContainer` as well?

Reply:
yep, it exists - on my machine. Simply hadn't staged it for commit.